cmd/genji/shell/shell.go

```diff
@@ -44,6 +45,11 @@ type Options struct {
 	Engine string
 	// Path of the database file or directory that will be created.
 	DBPath string
+	// Table insertion table for insert command.
```

Comment:
The shell type is owned by the root command, this is the only command that runs the genji shell.
The insert command shouldn't use the shell at all, it must simply open the selected db, run a bunch of inserts and quit.

Reply:
I see but how to manage the json from stdin?
``` 
$>curl https://api.github.com/repos/genjidb/genji/issues | genji insert -f my.db -t foo
```

Reply:
Reading from stdin is not limited to the Shell type, you can read from stdin yourself. Here is an example: https://sourcegraph.com/github.com/genjidb/genji@master/-/blob/cmd/genji/shell/shell.go?utm_source=VSCode-1.1.0#L304-314

Reply:
This is marked as resolved by wasn't applied. Please only resolve conversations when they are applied to the PR

Reply:
I just check if it is in a pipe with the following
```go

        fi, _ := os.Stdin.Stat()
	m := fi.Mode()
	//Insert command is given in the pipe
	if (m & os.ModeNamedPipe) != 0 {
		return executeInsertCommand(db, table, os.Stdin)
	}
```